apps/src/gamelab/GameLabVisualizationColumn.jsx

```diff
@@ -67,6 +78,11 @@ class GameLabVisualizationColumn extends React.Component {
     }
   };
```

Comment:
Same feedback for unifying `pickerTouchEnd` / `pickerMouseUp` — do we need to grab the latest offset coordinate if we're already updating the picker on mouse move?